### ISSUE TITLE
Failed to publish dts-gen@0.5.2?

### ISSUE THREAD

**Issue by mizunashi-mana** [Microsoft/dts-gen#38]
Missing template option

```
$ npm i dts-gen@0.5.2
$ npm ls dts-gen
test-ts@1.0.0 ~/test-ts
└── dts-gen@0.5.2 

$ cat node_modules/dts-gen/bin/lib/run.js | head -25
#!/usr/bin/env node
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var guess = require("./");
var yargs = require("yargs");
var fs = require("fs");
var path = require("path");
var args = yargs
    .alias('m', 'module')
    .alias('i', 'identifier')
    .alias('e', 'expression')
    .alias('n', 'name')
    .alias('f', 'file')
    .alias('d', 'dt')
    .alias('s', 'stdout')
    .alias('o', 'overwrite')
    .argv;
var ArgsError = (function (_super) {
    __extends(ArgsError, _super);
    function ArgsError(argsError) {
        var _this = _super.call(this) || this;
```

**Comment by jasperjn** [comment]
Any update?
The package published on npm is still outdated.

**Comment by RyanCavanaugh** [comment]
Published 0.5.3
